### PR TITLE
Removed warnings from environment config

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Removes warnings from environment config.
Adds copying of warnings from `InternalEnvironment` to `Runtime`

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7495

### CODE REVIEW

**Comment by sleshchenko** [comment]
@ashumilova Please review my changes in dashboard https://github.com/eclipse/che/pull/7663/commits/5ad68646e9be289a21b978e872e37103ebc7cd1b

**Comment by sleshchenko** [comment]
ci-test

File: dashboard/src/app/workspaces/workspace-details/warnings/workspace-warnings.controller.ts [hunk 1]
```diff
@@ -33,9 +33,6 @@ export class WorkspaceWarningsController {
     this.warnings = [];
 
     let environment = (this.workspace && this.workspace.config && this.workspace.config.defaultEnv) ? this.workspace.config.environments[this.workspace.config.defaultEnv] : null;
-    if (environment) {
-      this.warnings = this.warnings.concat(environment.warnings);
-    }
 
     if (this.workspace && this.workspace.runtime) {
       this.warnings = this.warnings.concat(this.workspace.runtime.warnings);
```
Comment (benoitf) [inline dashboard/src/app/workspaces/workspace-details/warnings/workspace-warnings.controller.ts]: do we still have warnings in workspace.runtime ?

Comment (sleshchenko) [reply dashboard/src/app/workspaces/workspace-details/warnings/workspace-warnings.controller.ts]: yes

Comment (ashumilova) [reply dashboard/src/app/workspaces/workspace-details/warnings/workspace-warnings.controller.ts]: then environment variable could be removed https://github.com/eclipse/che/pull/7663/files#diff-96c3dcf163c4af0310e5640e7e88ecc4L35

Comment (ashumilova) [reply dashboard/src/app/workspaces/workspace-details/warnings/workspace-warnings.controller.ts]: and mentioning about environment in comments https://github.com/sleshchenko/che/blob/5f2f76bb9a54abd543eb9df3b7e007ddd5ed9e23/dashboard/src/app/workspaces/workspace-details/warnings/workspace-warnings.controller.ts#L15

**Review by ashumilova — APPROVED** [review]

File: wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/spi/InternalRuntimeTest.java [hunk 2]
```diff
@@ -582,13 +583,14 @@ public TestInternalRuntime(URLRewriter urlRewriter, boolean running)
                                                  emptyMap(),
                                                  null),
                              new RuntimeIdentityImpl("ws", "env", "owner"),
-                             null,
+                              null,
```
Comment (garagatyi) [inline wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/spi/InternalRuntimeTest.java]: Can you remove this commented code?

**Review by garagatyi — APPROVED** [review]

File: wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/spi/InternalRuntimeTest.java (hunk 2 above)
Comment (sleshchenko) [reply wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/spi/InternalRuntimeTest.java]: I think yes, I can.

**Pull request merged**